### PR TITLE
Topcorr function and tests 

### PR DESCRIPTION
**Pull request by doraqmon**

### CODE REVIEW

File: tests/test_topcorr.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  result_1 = topcorr.topcorr(test_df)
+  assert result_1.shape == (1, 3)
+  assert result_1.equals(pd.DataFrame({'Feature 1': ['y'], 'Feature 2': ['x'], 'Absolute Correlation' : [1.0]}))
+  
+  # test when k is not default and equal to number of feature pairs
+  result_2 = topcorr.topcorr(test_df)
+  assert result_2.shape == (1, 3)
+  assert result_2.equals(pd.DataFrame({'Feature 1': ['y'], 'Feature 2': ['x'], 'Absolute Correlation' : [1.0]}))
+  
+  # test when k is not default and less than number of feature pairs
```
Comment (HanyingZhang) [inline tests/test_topcorr.py:50]: there is no test explicitly indicated to show the sorting functionality. Maybe you can improve this one to show that all the correlations are sorted?

Comment (doraqmon) [reply tests/test_topcorr.py:50]: Hi, I checked the output is correct or not , which will show my sorting functionality is tested.

Comment (doraqmon) [reply tests/test_topcorr.py:50]: I rethought about this. you are correct.
I will add another test_input which does not only have a correlation as 1 to test the sorting functionality!

File: pythcat/topcorr.py [hunk 2]
```diff
@@ -1,16 +1,52 @@
...
-    """
+  Returns
+  -------
+  pandas.core.frame.DataFrame
+    The dataframe of top k correlated features
+  
+  Examples
+  --------
+  >>> df = pd.DataFrame({'x': [1, 2], 'y': [3, 4]})
+  >>> topcorr(df, 1)
```
Comment (HanyingZhang) [inline pythcat/topcorr.py:22]: Maybe you can add the output in the example as:
```suggestion
  >>> topcorr(df, 1)
  Feature 1 | Feature 2 | Absolute Correlation
  -- | -- | --
  y  | x  | 1.0
```

**Review by HanyingZhang — CHANGES_REQUESTED** [review]

File: tests/test_topcorr.py (hunk 1 above)
Comment (doraqmon) [reply tests/test_topcorr.py:50]: I have modified the last test case based on your suggestion

**Review by HanyingZhang — APPROVED** [review]

**Pull request merged**